docs/api-reference/1_5/documents/_overview.md

```diff
@@ -0,0 +1,92 @@
+# <strong>API OVERVIEW</strong>
+
+Welcome to the Kubernetes API.  You can use our API to access Kubernetes API endpoints to read and write Kubernetes
+resource objects.
+
+## Resource Categories
+
+This is a highlevel overview of the basic types of resources provide by the Kubernetes API and their primary functions.
+
+**Workloads** *Run your containers*: These are responsible for managing and running your containers on the cluster.
+
+**Discovery & LB Resources** *Access your containers*: These are responsible for stitching your workloads together into an externally accessible Loadbalanced Service.
```

Comment:
**Discovery and Load-Balancing** resources are objects you use to "stitch" your workloads together into an externally accessible, load-balanced Service.